main.go

```diff
@@ -15,13 +15,14 @@ func main() {
 	provider := getInstanceOfAzureRMProvider()
 
 	// Example talking to the Azure resources using the provider
-	useProviderToTalkToAzure(provider)
+	configureProvider(provider)
+	// exampleChangesToResourceGroup(provider)
 
 	// Example creating CRDs in K8s with correct structure based on TF Schemas
-	createCRDsForResources(provider)
+	go createCRDsForResources(provider)
```

Comment:
I assume that the informer doesn't require these to have been created in order to be able to set up the watch?

Reply:
I *thought* it didn’t but I’ve not tested that theory. Looking at it now I feel like I’m probably wrong and the informer will fail of the types aren’t registered